package.json

```diff
@@ -23,26 +23,37 @@
     "url": "https://github.com/apache/daffodil-vscode/issues"
   },
   "scripts": {
+    "omega-edit-download": "tsc -p ./ && node -e 'require(\"./out/omega_edit/download.js\").downloadServer()'",
     "vscode:prepublish": "yarn run package-ext",
     "precompile": "node -p \"'export const LIB_VERSION = ' + JSON.stringify(require('./package.json').version) + ';'\" > src/version.ts",
-    "compile": "tsc -p ./",
+    "compile": "tsc -p ./ && node out/omega_edit/download.js",
     "lint": "yarn run prettier src -c",
-    "watch": "webpack --watch --devtool nosources-source-map --info-verbosity verbose --config ./build/extension.webpack.config.js",
+    "watch": "yarn omega-edit-download && webpack --watch --devtool nosources-source-map --config ./build/extension.webpack.config.js",
     "watch2": "tsc -watch -p ./",
     "package": "vsce package",
     "publish": "vsce publish",
     "package-ext": "webpack --mode production --config ./build/extension.webpack.config.js",
     "pretest": "yarn run compile",
     "test": "node node_modules/mocha/bin/_mocha -u tdd --timeout 999999 --colors ./out/tests",
     "sbt": "sbt universal:packageBin",
-    "pre-build": "mv LICENSE tmp.LICENSE && mv NOTICE tmp.NOTICE && cp build/bin.LICENSE LICENSE && cp build/bin.NOTICE NOTICE",
+    "pre-build": "run-script-os",
+    "pre-build:default": "mv LICENSE tmp.LICENSE && mv NOTICE tmp.NOTICE && cp build/bin.LICENSE LICENSE && cp build/bin.NOTICE NOTICE",
+    "pre-build:windows": "move LICENSE tmp.LICENSE && move NOTICE tmp.NOTICE && copy build/bin.LICENSE LICENSE && copy build/bin.NOTICE NOTICE",
     "build": "yarn pre-build && yarn sbt && yarn install && yarn compile && yarn package && yarn post-build",
-    "post-build": "rm -rf NOTICE LICENSE && mv tmp.LICENSE LICENSE && mv tmp.NOTICE NOTICE"
+    "post-build": "run-script-os",
+    "post-build:default": "rm NOTICE && rm LICENSE && mv tmp.LICENSE LICENSE && mv tmp.NOTICE NOTICE",
+    "post-build:windows": "del /q NOTICE && del /q LICENSE && move tmp.LICENSE LICENSE && move tmp.NOTICE NOTICE"
   },
   "dependencies": {
+    "@grpc/grpc-js": "1.5.4",
+    "@types/moo": "^0.5.5",
     "await-notify": "1.0.1",
     "child_process": "^1.0.2",
+    "google-protobuf": "^3.19.4",
     "hexy": "^0.3.1",
+    "moo": "^0.5.1",
+    "omega-edit": "0.9.8",
+    "run-script-os": "^1.1.6",
```

Comment:
@stevedlawrence I am not sure what to do here as removing `^`'s in most places cause things to break.

Reply:
My understand is the `^` just means any compatible version. So we might not necessarily get 1.1.6, we could get 1.1.7 or 1.2.0 or something (I'm not sure how npm determines compatibility).

I'm not sure why specific versions aren't working .It might mean there are some conflicting dependencies and they all need to be updated to the latest version?

Reply:
@stevedlawrence So I was able to remove all `^` from dependencies that aren't `@types`. Removing it from these caused error in the CI so they were kept. This needs to be done because we technically support multiple different version of node not just 14 so we need those specific items to be allowed to be a range of version so it works for different versions of node.